include/json/reader.h

```diff
@@ -262,16 +270,34 @@ class JSON_API CharReader {
    * error occurred.
    */
   virtual bool parse(char const* beginDoc, char const* endDoc, Value* root,
-                     String* errs) = 0;
+                     String* errs) final;
+
+   /** \brief Returns a vector of structured errors encountered while parsing.
```

Comment:
This doc should answer some questions about the way CharReader stores the errors after a parse.

Does each parse call reset the stored errors vector?
Does a call to this function consume and reset the errors?